jvm-runtime/ftl-runtime/common/runtime/src/test/java/xyz/block/ftl/runtime/Animal.java

```diff
@@ -0,0 +1,17 @@
+package xyz.block.ftl.runtime;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import xyz.block.ftl.Enum;
+
+@Enum
+public interface Animal {
+    @JsonIgnore
```

Comment:
I wonder if we can eventually remove the need for these annotations.

Reply:
Yeah, maybe we can customise the serialisation further to ignore them